### ISSUE TITLE
Putting a different head on a human body and its causes

### ISSUE THREAD

**Issue by Brolaire** [d3athrow/vgstation13#9026]
I started off on the test server with a monkeyman and a farwacatbeast and cut off their heads in the surgery room. (The green head is the human one, called Frank Taylor and the grey head is the catbeast one, titled Simon Whittier).
![image](https://cloud.githubusercontent.com/assets/11573250/14034386/1e841208-f223-11e5-83c7-c58e11c3c79a.png)

After that, I put the catbeast head on the human, and out came out a human. After reconstructing his face, you can see it's actually Frank Taylor, and not Simon Whittier as expected.
![image](https://cloud.githubusercontent.com/assets/11573250/14034846/81ef0fc4-f227-11e5-8b68-cd62ff136ff1.png)

After removing the heads, it's the same heads again, but the damage overlay sprite is still on the headless body, it wasn't before.
I don't know what I would suggest for inter-species head transplanting. Maybe make it not possible? This would make ordering monkey-like animals or brain transplant surgery more interesting over "forcefully behead corpse and slap it on monkeyman body".
That would also mean we need plasmamonkeys and grey monkeys.

Besides that, I think it would make more sense to tie the identity to the head instead of the body, if that is possible.

**Comment by Exxion** [comment]
Making transplanted limbs/heads look like they should _might_ be worth stealing from Goon

"Might" because there's a good chance it will either be harder to port than to code or they've done it in in a bad way

**Comment by Brolaire** [comment]
Not familiar with how Goon does it, what's different?

**Comment by Exxion** [comment]
Well, the primary difference is that it actually works
Like the different arms, legs, etc. actually look different on your character
Actually, now that I think about it, our robotic limbs already act like that, just Goon has more types

If you mean codewise, I don't know. Haven't looked.

**Comment by gbasood** [comment]
> stealing from goon

No.
Code it from scratch.
Their organs system is very different from ours. Steal the concept maybe, not the code.

**Comment by Exxion** [comment]
Sorry, I figured it would go without saying that when I say to steal things from Goon, I pretty much always mean concept

**Comment by Brolaire** [comment]
> /labels/honk

Why?

**Comment by gbasood** [comment]
Because it's a silly bug
I'm sure someone will come in later and remove it don't worry

**Comment by Brolaire** [comment]
N E V E R E V E R

**Comment by PJB3005** [comment]
@Exxion or steal it from Bay...

**Comment by Exxion** [comment]
Does Bay have that working?
Well then let's get stealing

**Comment by PJB3005** [comment]
No Bay completely reworked organs and made them awesome.

**Comment by Exxion** [comment]
That sounds good
And it should be FAR easier to port from Bay than Goon

**Comment by Shadowmech88** [comment]
I'm not really sure this is a "bug" per se. I think the system is working as intended, it's just a bad system, what with tying identity to the torso and all that.
I'm going to relabel this to Feature Request.

**Comment by gbasood** [comment]
p.s. someone really should make identity not part of torso already

**Comment by Exxion** [comment]
Yes, but nobody *will* because it would involve a major rewrite of lifecode
The torso is the mob, so obviously any variables that belong to the mob that are not explicitly moved to the head will remain on the torso

**Comment by Shadowmech88** [comment]
Sounds like a new project for the Antag Datums team.
